tools/client-plugins/gatsby-source-challenges/create-challenge-nodes.js

```diff
@@ -7,27 +7,10 @@ function createChallengeNode(
   reporter,
   { isReloading } = { isReloading: false }
 ) {
-  // challengeType 11 is for video challenges (they only have instructions)
-  // challengeType 7 is for certificates (they only have tests)
-  // challengeType 12 is for CodeAlly/CodeRoad challenge
+  if (!challenge.description) challenge.description = '';
+  if (!challenge.instructions) challenge.instructions = '';
+  if (!challenge.seed) challenge.seed = '';
```

Comment:
Sorry to be a pain, but this belongs in the `generateChallengeCreator` function in get-challenges. That sets up the challenge, so it's better not to spread the responsibility around.

Also, the output of get-challenges goes through the challenge schema, so we can validate that these properties are defined.